src/test/compile-fail/rfc-2126-extern-in-paths/single-segment.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// aux-build:xcrate.rs
+
+#![feature(extern_in_paths)]
+
+use extern; //~ ERROR unresolved import `extern`
+            //~^ NOTE no `extern` in the root
+use extern::*; //~ ERROR unresolved import `extern::*`
+               //~^ NOTE cannot glob-import all possible crates
+
+fn main() {
+    let s = extern::xcrate; //~ ERROR expected value, found module `extern::xcrate`
```

Comment:
Is there a reason not to support `extern::` paths in this location? I had thought that one of the bonus points to using a syntax like `extern` for absolute paths was precisely that it could be unambiguously used anywhere. Basically, the set of paths accepted in a `use` would now be a strict subset of those that could be accepted elsewhere (excluding relative paths, obviously).

Reply:
It's accepted in this location (as tested in the `run-pass` test), here we are just checking that `extern::xcrate` is indeed a crate, and a crate can't be used as a value.

Reply:
oh oh I see =) thanks